source/settings/plugin/expire-plugin.rst

```diff
@@ -4,6 +4,8 @@
 expire plugin
 ===============
 
+.. versionremoved:: 2.3.14 Use autoexpunge namespace mailbox setting instead.
```

Comment:
Link to https://doc.dovecot.org/configuration_manual/namespace/#mailbox-settings or maybe add a new #autoexpunge anchor